Sources/PointFree/Account/PrivateRss.swift

```diff
@@ -0,0 +1,255 @@
+import Foundation
+import Html
+import HttpPipeline
+import Optics
+import Prelude
+import Tuple
+
+let accountRssMiddleware: Middleware<StatusLineOpen, ResponseEnded, Tuple2<Database.User.Id, Database.User.RssSalt>, Data> =
+{ fetchUser >=> $0 }
```

Comment:
remember this trick, it's how you can lift a middleware into a middleware transformer.

Reply:
Yup. Still confusing to me each time. Wish I could internalize it!